.pre-commit-config.yaml

```diff
@@ -0,0 +1,35 @@
+fail_fast: true
+repos:
+  -
+    repo: https://github.com/pre-commit/pre-commit-hooks
+    rev: v2.0.0
+    hooks:
+      - id: check-added-large-files
+      - id: check-case-conflict
+      - id: check-merge-conflict
+      - id: check-json
+      - id: check-yaml
+      - id: detect-aws-credentials
+        args: ["--allow-missing-credentials"]
+      - id: check-vcs-permalinks
+  -
+    repo: https://github.com/jumanjihouse/pre-commit-hooks
+    rev: 1.10.2
+    hooks:
+      - id: require-ascii
+      - id: shellcheck
+
+  -
+    repo: https://github.com/igorshubovych/markdownlint-cli
+    rev: v0.13.0
```

Comment:
Latest is 0.18.0, not sure how much they did break

Reply:
Good catch, fixed!